tests/lib-bench/src/miniboxing/benchmarks/rrbvector/miniboxed/MbLLSRegression.scala

```diff
@@ -0,0 +1,48 @@
+package miniboxing.benchmarks.rrbvector.miniboxed
+
+import miniboxing.runtime.math.MiniboxedFractional
+
+// Learn Least Square Regression using RRBVector
+class MbLLSRegression[@miniboxed A](val x: RRBVector[A], val y: RRBVector[A])(implicit numeric: MiniboxedFractional[A]) {
+  def sum_x: A = x.sum
+  def sum_y: A = y.sum
+  def sum_xy: A = {
+    var sum = numeric.zero
+    var i = 0
+    while (i < x.length) {
+      sum = numeric.plus(sum, numeric.times(x(i), y(i)))
+      i = i + 1
+    }
+    sum
+  }
```

Comment:
Numbers with this change:
``miniboxed :  2262.81464 +/- 41.23587``
``generic :  2533.86285 +/-  64.59214``